src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.Shell/Pages/SettingsPage.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CmdPal.Ext.Shell.Helpers;
+using Microsoft.CmdPal.Extensions;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+namespace Microsoft.CmdPal.Ext.Shell.Pages;
+
+internal sealed partial class SettingsPage : FormPage
+{
+    private readonly Settings _settings;
+    private readonly SettingsManager _settingsManager;
+
+    public override IForm[] Forms()
+    {
+        var s = _settings.ToForms();
+        return s;
+    }
+
+    public SettingsPage(SettingsManager settingsManager)
+    {
+        Name = "Test Settings Page"; // TODO --> localization
```

Comment:
🫨-- thought: should we have the same footer thing we were working on from before so we can just call this 'Settings' for each of the built-in-extensions but it will be clear to the user like which extension is providing that page???

Reply:
#194 for the thought above